src/App.jsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react'
+import reactLogo from './assets/react.svg'
+import './App.css'
+import { ChemicalCard } from './ChemicalCard.jsx'
+import { Dialog } from 'primereact/dialog';
+import { Button } from 'primereact/button';
+
+import "primereact/resources/themes/lara-light-indigo/theme.css";  //theme
+import "primereact/resources/primereact.min.css";                  //core css
+import "primeicons/primeicons.css";                                //icons
+import "primeflex/primeflex.css";                                  //easy CSS flex
+
+
+// Card: https://www.primefaces.org/primereact/card/
+// Grid system responsive at bottom of: https://www.primefaces.org/primeflex/gridsystem
+// Update list of components: https://beta.reactjs.org/reference/react/useState#examples-objects
+
+function App() {
+  const [count, setCount] = useState(0)
+  const [state, setState] = useState(false)
+
+  return (
+    <div className="App">
+      <h1>Happy hormone boosters</h1>
+      <div class="grid">
+          <div class="col-12 lg:col-6"><ChemicalCard chemical="Dopamin" description='The reward chemical' activities={['Completing a task', 'Doing self-care activities', 'Eating food', 'Celebrating little wins']} color='PapayaWhip'/></div>
+          <div class="col-12 lg:col-6"><ChemicalCard chemical="Oxytocin" description='The love hormone' activities={['Playing with a dog', 'Playing with a baby', 'Holding hands', 'Hugging your family', 'Giving compliments']} color='LightCyan'/></div>
+          <div class="col-12 lg:col-6"><ChemicalCard chemical="Serotonin" description='The mood stabilizer' activities={['Meditating', 'Running', 'Sun exposure', 'Walk in nature', 'Swimming', 'Cycling']} color='HoneyDew'/></div>
+          <div class="col-12 lg:col-6"><ChemicalCard chemical="Endorphin" description='The pain killer' activities={['Laughter exercises', 'Essential oils', 'Watch a comedy', 'Dark chocolate', 'Exercising']} color='LightYellow'/></div>
```

Comment:
Oh! For static data like this something I like to do is put it in a JSON file of its own. Then you can import that file `import data from './static.json'` and render the components dynamically:

```tsx
<div className="grid">
  {data.activities.map((a) => <ChemicalCard {...a} />)}
</div>
```

You could also skip the json & just put it in a js object like this:

```ts
const DATA [/* data here */];
```
Makes editing a little easier 😄 

Also, I'm not sure if they've changed it but `class` had to be `className` in react traditionally (because `class` is a reserved work in js). I think this has changed but good to be consistant.

Reply:
I thought about it but didn't know how to do it cleanly. I went for the JSON file option because it looks more like real data this way!
Thanks!